src/ripple/server/impl/ServerHandlerImp.cpp

```diff
@@ -521,7 +520,7 @@ struct ParsedPort
 
     boost::optional<boost::asio::ip::address> ip;
     boost::optional<std::uint16_t> port;
-    boost::optional<bool> allow_admin;
+    boost::optional<std::vector<beast::IP::Address>> admin_ip;
```

Comment:
Does it make sense to remove the `boost::optional` and just use `std::vector`? I don't see in our code where a `false` boost optional here would act differently from an empty vector (tho I could have missed it).